### PR TITLE
Annotator support prep

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

File: docs/system/setup.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+External large files should be provisioned by running the ansible tool:
+
+    ansible-playbook -i ansible/hosts.yml ansible/provision_files.yml
+
+This may include large files for running the system, like ML model files, or
+other files required for building docker images.
+
+This provisioning may require additional configuration and variables set in
+your environment in order to satisfy some permissions:
+* ``GIRDER_API_KEY`` will need to be set in order to acquire protected files from
```
Comment (hdefazio) [inline docs/system/setup.rst:46]: Add instructions on where to find this? I had a version of that in a google doc

File: docs/system/setup.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  ``data.kitware.com``.
+
+The configuration that controls what is staged and where is located
+in the ``ansible/roles/provision-files/vars/main.yml`` file.
+
+
+HL2 First-time Setup
+====================
+
+* User account creation
```
Comment (hdefazio) [inline docs/system/setup.rst:56]: Link to instructions if we have them in the repo already?

File: docs/system/setup.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* docker-compose
+
+You will want to add your user to the ``docker`` group to facilitate using
+docker commands.
+
+The network interface to which we are going to connect to the HoloLens2 should
+be appropriately configured:
+
+* IPv4 should be set to 192.168.1.100 (address), 255.255.255.0 (netmask), and
+  192.168.1.1 (gateway)
```
Comment (hdefazio) [inline docs/system/setup.rst:22]: This is repeated down below in "HL2 First-time Setup"

**Comment by hdefazio** [comment]
Setup guide: 
https://docs.google.com/document/d/1xKN9IOaF7xQXzk4oiKs-YxnSnsgLuTUKPKd-6D6q-SQ/edit#heading=h.vtjaolp500s3

Notes: https://docs.google.com/document/d/1cql3RFOgzI3Cf5CX4j646Uj6VGMg9v1jBcWFhQ0AHhA/edit#heading=h.xqppqqq8sesf

**Comment by hdefazio** [comment]
Link to these in the data collection doc: https://docs.google.com/document/d/1kMoN7ciIn4u-D4I5e7v6hXuRAxN5gz8AT8HNGR7roWc/edit#heading=h.63wavuv60cor

File: docs/system/setup.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+===================
+The target linux machine will need to have installed:
+
+* ansible
+* docker
+* nvidia-docker2
+* docker-compose
+
+You will want to add your user to the ``docker`` group to facilitate using
+docker commands.
```
[the remark this answers is not in the document]
Comment (Purg) [reply docs/system/setup.rst:16]: Adding a subsection to provide the `usermod` command.

File: docs/system/setup.rst (hunk 2 above)
Comment (Purg) [reply docs/system/setup.rst:66]: We don't currently. This is covered by MS documentation. Will add the link to [this](https://learn.microsoft.com/en-us/hololens/hololens2-start) here.

File: docs/system/setup.rst (hunk 3 above)
Comment (Purg) [reply docs/system/setup.rst:23]: This is slightly different: this describes using the host static IP to end if **`100`** while the HL2 section below describes the static IP to be **`101`**.

File: docs/system/setup.rst (hunk 1 above)
Comment (Purg) [reply docs/system/setup.rst:56]: Adding a subsection for this.

**Comment by Purg** [comment]
I have also updated https://docs.google.com/document/d/1kMoN7ciIn4u-D4I5e7v6hXuRAxN5gz8AT8HNGR7roWc/edit#heading=h.1wbuk1tksg5l

**Comment by hdefazio** [comment]
LGTM

**Pull request merged**